windows/micropython.vcxproj

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project DefaultTargets="Build" ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
This, along with PlatformToolset, will make the project file unusable for earlier versions of VS (i.e. 2013). I knew the day would come we'd need to take care of dealing with muliple versions of VS (which is why the meat of the configuration is already in seperate .props files and the project is just a 'container' now) but never put much thought into how to do this exactly. Do you have any suggestions?